packages/smarthr-ui/src/components/Dialog/ActionDialog/ActionDialogContentInner.tsx

```diff
@@ -99,8 +98,9 @@ export const ActionDialogContentInner: FC<ActionDialogContentInnerProps> = ({
   }, [])
 
   return (
-    // eslint-disable-next-line smarthr/a11y-heading-in-sectioning-content
-    <Section className={styles.wrapper}>
+    // スクロールできるようにするため、Stack[as="section"]を使用
```

Comment:
コメントをよりわかりやすく、Section使うなよ！ってことがわかるようにしたほうが良さそうです

```suggestion
    // HINT: bodyをスクロールできるようにするため、Sectionではなく、Stack[as="section"][gap=0]を使用する必要がある
```